pkg/graph/resolvers/plan_favorite.go

```diff
@@ -0,0 +1,66 @@
+package resolvers
+
+import (
+	"go.uber.org/zap"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/mint-app/pkg/authentication"
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/storage"
+)
+
+//PlanFavoriteCollection returns a list of favorites
+func PlanFavoriteCollection(logger *zap.Logger, principal authentication.Principal, store *storage.Store) ([]*models.PlanFavorite, error) {
+	favorites, err := store.PlanFavoriteCollectionByUser(logger, principal.ID(), false)
+	if err != nil {
+		return nil, err
+	}
+
+	return favorites, err
+}
+
+//IsPlanFavorited checks if a model plan is a favorite.
+func IsPlanFavorited(logger *zap.Logger, principal authentication.Principal, store *storage.Store, modelPlandID uuid.UUID) (bool, error) {
```

Comment:
```suggestion
func IsPlanFavorited(logger *zap.Logger, principal authentication.Principal, store *storage.Store, modelPlanID uuid.UUID) (bool, error) {
```

Just a typo in a name here!